vscode/src/common.ts

```diff
@@ -37,3 +38,17 @@ export function toVscodeRange(range: IRange): Range {
 export function toVscodeLocation(location: ILocation): any {
   return new Location(Uri.parse(location.source), toVscodeRange(location.span));
 }
+
+export function toVscodeWorkspaceEdit(
```

Comment:
I like this! I'd love it if you went a step further and refactored `rename.ts` to call into this as well when it's constructing `WorkspaceEdit`s.

Reply:
Done!